django/contrib/messages/storage/cookie.py

```diff
@@ -59,6 +61,7 @@ class CookieStorage(BaseStorage):
     # restrict the session cookie to 1/2 of 4kb. See #18781.
     max_cookie_size = 2048
     not_finished = '__messagesnotfinished__'
+    hash_algorithm = 'blake2b'
```

Comment:
I note the ticket referred to moving toward `sha256`, and this is using `blake2b`.

Any reason for this choice? (Not objecting, just wondering.)

Reply:
I personally do not see any reason not to use `blake2b`, it is in `hashlib.algorithms_guaranteed` since 3.6 so we should be fine. /cc @ubernostrum